src/anim/evaluator/anim-evaluator.js

```diff
@@ -238,6 +236,16 @@ class AnimEvaluator {
         return null;
     }
 
+    rebind() {
+        this._binder.rebind();
+        this._targets = {};
+        var clips = [...this.clips];
```

Comment:
please make sure this transpiles to something acceptable in es5 .. I don't think we've tested it yet.

Reply:
Yep i've tested this using `rollup -c --watch --environment target:es5`